charts/rancher-logging-example/values.yaml

```diff
@@ -55,4 +60,19 @@ elasticsearch:
     enabled: false
   replicas: 1
   minimumMasterNodes: 1
+  nodeSelector:
+    kubernetes.io/os: linux
+  tolerations:
+    - key: cattle.io/os
+      operator: "Equal"
+      value: "linux"
+      effect: NoSchedule
+  # Uncomment the following lines to remove resource limits and requests.
```

Comment:
do we need these comments?

Reply:
We don't _need_ them, but I've seen other charts do this. What do you think?

Reply:
They were not included in this chart because we set limits and resources for the user.  Given how constant the load is on the pod I we set them for the user.

Reply:
Ok great. Then I'll remove them since they are in the README anyway.